decidim-accountability/app/forms/decidim/accountability/admin/result_form.rb

```diff
@@ -39,8 +39,15 @@ def map_model(model)
           self.decidim_category_id = model.category.try(:id)
         end
 
+        def proposal_objs
```

Comment:
Can you change this method name? I don't understand what `proposal_objs` mean.

Maybe this should be called `proposals`, and the next method be called `proposal_names_and_ids`, or similar?

Reply:
I've removed former #proposals method (it wasn't being used!) and renamed `proposal_objs` to `proposals`.